packages/react-components/src/organisms/TeamCard.tsx

```diff
@@ -38,24 +51,30 @@ type TeamCardProps = Pick<
 const TeamCard: React.FC<TeamCardProps> = ({
   id,
   displayName,
+  inactiveSince,
   projectTitle,
   expertiseAndResourceTags,
   members,
   labCount,
 }) => (
-  <Card>
-    <LinkHeadline
-      level={2}
-      styleAsHeading={4}
-      href={network({}).teams({}).team({ teamId: id }).$}
-    >
-      Team {displayName}
-    </LinkHeadline>
+  <Card accent={inactiveSince ? 'neutral200' : 'default'}>
+    <div css={titleStyle}>
+      <LinkHeadline
+        level={2}
+        styleAsHeading={4}
+        href={network({}).teams({}).team({ teamId: id }).$}
+      >
+        Team {displayName}
+      </LinkHeadline>
+      {!!inactiveSince && <StateTag label="Inactive" />}
+    </div>
     <Anchor href={network({}).teams({}).team({ teamId: id }).$}>
-      <Paragraph accent="lead">{projectTitle}</Paragraph>
+      <Paragraph hasMargin={false} accent="lead">
```

Comment:
This paragraph has margin vertical 12px if we don't pass `hasMargin={false}` so the distance between the team title and project proposal became 24px (LinkHeadline h2 also has margin vertical 12px). So I added this to avoid adding a negative margin to titleStyle